calico-enterprise/reference/installation/_api.mdx

```diff
@@ -17677,7 +17677,7 @@ string
 
 <p>
 Name is an enum which identifies the Manager Deployment container by name.
-Supported values are: tigera-voltron, tigera-manager, tigera-es-proxy
+Supported values are: tigera-voltron, tigera-manager, tigera-ui-apis
```

Comment:
This is auto generated code, needs to be updated after https://github.com/tigera/operator/pull/3531 is merged